sale_order_archive/readme/INSTALL.rst

```diff
@@ -0,0 +1 @@
+You need to install *sale_management* module for accessing the needed menus.
```

Comment:
Why don't we add sale_management module in dependency as user will have to install that module?